### PR TITLE
Move MCT component to TFX Addons

### PR DESCRIPTION
**Pull request by hanneshapke**
Addresses #[82](https://github.com/tensorflow/tfx-addons/issues/82)

- [x] Tests pass
- [x] Test example notebook
- [x] Update `CODEOWNER`

### CODE REVIEW

**Comment by hanneshapke** [comment]
@codesue Let's chat async what to do with https://github.com/tensorflow/model-card-toolkit/blob/master/model_card_toolkit/utils/tfx_util.py and the related tests. Are there reasons for it to stay in the MCT repo?

**Comment by hanneshapke** [comment]
I think it should probably stay. It is used in core and it seems to contain the connector pieces to the TFX libraries, not the component par se.

File: examples/model_card_generator/MLMD_Model_Card_Toolkit_Demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,1235 @@
...
+        "# limitations under the License."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "23R0Z9RojXYW"
+      },
+      "source": [
+        "# MLMD Model Card Toolkit Demo"
```
Comment (codesue) [inline examples/model_card_generator/MLMD_Model_Card_Toolkit_Demo.ipynb:40]: I believe MLMD is only used indirectly through TFX in this example. Consider removing the MLMD import and version check and removing mentions of MLMD wherever it makes sense to do so.

File: examples/model_card_generator/MLMD_Model_Card_Toolkit_Demo.ipynb [hunk 2]
```diff
@@ -0,0 +1,1235 @@
...
+      ],
+      "metadata": {
+        "id": "l8LY5bMUJsuM"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import json\n",
+        "import model_card_toolkit as mctlib\n",
```
Comment (codesue) [inline examples/model_card_generator/MLMD_Model_Card_Toolkit_Demo.ipynb:1104]: Let's remove this unused import.

File: examples/model_card_generator/census_income_constants.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+
```
Comment (codesue) [inline examples/model_card_generator/census_income_constants.py:1]: You were right about these files not being needed to run the example. I'm starting to think they might have been commited by mistake. We may want to consider adding them to the gitignore.

File: tfx_addons/model_card_generator/executor_test.py [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+        model_card_file_name = 'model_card.html'
+      self.assertIn(model_card_file_name, os.listdir(model_card_dir))
+      model_card_filepath = os.path.join(model_card_dir, model_card_file_name)
+      with open(model_card_filepath) as f:
+        model_card_content = f.read()
+      if exec_props:
+        self.assertEqual(model_card_content, 'hello world')
+      else:
+        self.assertStartsWith(model_card_content, '<!DOCTYPE html>')
+
```
Comment (codesue) [inline tfx_addons/model_card_generator/executor_test.py:149]: With https://github.com/tensorflow/model-card-toolkit/pull/251 merged, we can remove  `confidence_interval=model_card_pb2.ConfidenceInterval()` and `sensitive=model_card_pb2.SensitiveData()` below. See [example diff](https://github.com/tensorflow/model-card-toolkit/pull/251/commits/a515b86dca5817e6172fa9994614410448c368d6#diff-f7f73c3eb6f562cbe2ddb2770da515990e31a0a145e1c8c377fea59e5955566c).

File: examples/model_card_generator/census_income_constants.py (hunk 3 above)
Comment (hanneshapke) [reply examples/model_card_generator/census_income_constants.py]: I removed those notebook generated files

File: tfx_addons/model_card_generator/executor_test.py (hunk 4 above)
Comment (hanneshapke) [reply tfx_addons/model_card_generator/executor_test.py:149]: Thank you for the hint! I removed the references.

**Review by rcrowe-google — APPROVED** [review]
Thanks!
/lgtm

**Review by codesue — COMMENTED** [review]
lgtm after tests pass

File: .gitignore [hunk 5]
```diff
@@ -39,4 +39,9 @@ env/*
 
 #Editor
 .idea/*
-.vscode/*
\ No newline at end of file
+.vscode/*
+
+# unnecessary project files
```
[the remark this answers is not in the document]
Comment (hanneshapke) [reply .gitignore:44]: Good idea! Thank you!

File: CODEOWNERS [hunk 6]
```diff
@@ -61,6 +61,10 @@
 # PandasTransform Component
 /tfx_addons/pandas_transform @rcrowe-google
 
+# PandasTransform Component
+/tfx_addons/model_card_generator @codesue @hanneshapke
+/eamples/model_card_generator @codesue @hanneshapke
```
[the remark this answers is not in the document]
Comment (hanneshapke) [reply CODEOWNERS:66]: Arg, good catch.

File: .gitignore (hunk 5 above)
[the remark this answers is not in the document]
Comment (codesue) [reply .gitignore:44]: Cool! Let's revert the changes to this file before merging.

**Review by codesue — APPROVED** [review]

**Comment by hanneshapke** [comment]
Thank you @codesue

**Pull request merged**